### PR TITLE
gke firewall ports

### PR DESCRIPTION
**Pull request by so-jelly**
Describe issue affecting private GKE clusters and examples to investigate and resolve.

This is regarding:  https://github.com/istio/istio/issues/19532

### CODE REVIEW

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
+    For private GKE Clusters, the firewall rule created does not open ports (9443, 15017) needed by validation webhooks.
```
Comment (howardjohn) [inline content/en/docs/setup/platform-setup/gke/index.md]: nit: 9443 is for older versions, I think we only document the latest. You should (?) only need 15017 now.

Comment (so-jelly) [reply content/en/docs/setup/platform-setup/gke/index.md]: Updated to 1.5. Pilot Discover port.

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
+    For private GKE Clusters, the firewall rule created does not open a port (15017) needed by validation webhook.
```
Comment (Rachael-Graham) [inline content/en/docs/setup/platform-setup/gke/index.md]: lowercase clusters

maybe rephrase for easier translation: "the firewall rule that is created does not open the port (15017) needed by the validation webhook for master access."

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 3]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
...
+    
+    To review this firewall rule for master access:
+        {{< text bash >}}
+        $ gcloud compute firewall-rules list --filter="name~gke-<cluster-name>-[0-9a-z]\*-master" --format="value(name)"
+        {{< /text >}}
+    
+    This will replace existing rule:
```
Comment (Rachael-Graham) [inline content/en/docs/setup/platform-setup/gke/index.md]: To replace the existing rule and allow master access:

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 4]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
+    For private GKE Clusters, the firewall rule created does not open a port (15017) needed by validation webhook.
+    
+    To review this firewall rule for master access:
+        {{< text bash >}}
```
Comment (frankbu) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion

        {{< text bash >}}
```

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 5]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
...
+        $ gcloud compute firewall-rules list --filter="name~gke-<cluster-name>-[0-9a-z]\*-master" --format="value(name)"
+        {{< /text >}}
+    
+    This will replace existing rule:
+        {{< text bash >}}
```
Comment (frankbu) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion

        {{< text bash >}}
```

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 6]
```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
...
+        $ gcloud compute firewall-rules update <firewall-rule> --allow tcp:10250,tcp:443,tcp:15017
+        {{< /text >}}
```
Comment (frankbu) [inline content/en/docs/setup/platform-setup/gke/index.md]:
```suggestion
        {{< /text >}}

```

**Comment by so-jelly** [comment]
Sorry, first time contributing here, I'm not sure if my CLA is failing because of my name.   I updated all commits.
